### DISCUSSION TITLE
Spot Price

### DISCUSSION THREAD

**Discussion by edwapj** [amberelectric/public-api#47] [General]
Would it be worthwhile to adopt the NEM definition of the Spot Price and quote the Amber spotPerKwh price without GST ?

The Customer could then check the AEMO dashboard and be assured that the Amber API is delivering the authentic NEM spot prices for the right time periods and for the customer's own region.  
The decimal point will be different but the 4 digits will be the same.

**Comment by madpilot** [comment]
Don't you trust us? 😉 


This is a tricky one. By rights, it shouldn't have GST, but then that makes it different to the other prices in the result set, which will lead to confusion despite documentation. Should the other prices include GST? Normally the answer would be no, however the feedIn price has zero GST on it, so without that knowledge people will calculate bills incorrectly, hence us defaulting to GST inc.

As this will also be a breaking  change I'm going to defer the decision on this until we need to bump the version number of the API.

**Comment by madpilot** [comment]
I will talk this over with my colleges in the morning though.
